src/content/web3/index.md

```diff
@@ -51,6 +51,15 @@ Although it's challenging to provide a rigid definition of what Web3 is, a few c
 - **Web3 is permissionless:** everyone has equal access to participate in Web3, and no one gets excluded.
 - **Web3 has native payments:** it uses cryptocurrency for spending and sending money online instead of relying on the outdated infrastructure of banks and payment processors.
 - **Web3 is trustless:** it operates using incentives and economic mechanisms instead of relying on trusted third-parties.
+ 
+ ##  TABLE : COMPARISONS OF THE EVOLUTION OF THE WEB ##
+ 
+  |WEB 1.0 | WEB 2.0 | WEB 3.0 |
+  | ------ | ------ | ------ |
+  | Permissionless | Permissioned | Permissionless |
+  | Static content | Dynamic content | Dynamic content |
+  | Does not require complex server-side architectures. | Companies run your agreements on their servers. | It is an ownership economy. You own part of the protocols you interact with. Your agreements are run on decentralized networks. |
+  | Online directories, Static informational websites(e.g news), Text-based online forums. | Social networking(e.g Twitter, Meta, LinkedIn), E-commerce(e.g Amazon, Shopify), Collaborative platforms. | Crypto wallets(e.g MetaMask), Decentralized exchanges(e.g Binance, Uniswap), Decentralized tokens(e.g Bitcoin, NFTs), Decentraland [see here](https://www.bing.com/ck/a?!&&p=0be7e71f4b178550JmltdHM9MTY3ODA2MDgwMCZpZ3VpZD0zOWM1Njk2Yi00MGY3LTZjNjYtMTRmYi03YjE1NDRmNzYyOGUmaW5zaWQ9NTE5MQ&ptn=3&hsh=3&fclid=39c5696b-40f7-6c66-14fb-7b1544f7628e&psq=decentraland&u=a1aHR0cHM6Ly9kZWNlbnRyYWxhbmQub3JnLw&ntb=1)|
```

Comment:
Agree with @konopkja, would remove this Bing/Decentraland link. Would also suggest we highlight DEX's on Ethereum, and remove Binance from the examples.

Grammar notes: Need spaces between the list items and their examples, and `e.g.` should have two periods: "Social networking (e.g. Twitter, Meta, LinkedIn)".
Also, given our use of sentence casing throughout the site, would suggest only capitalizing the first word in each of these lists, unless it's a proper noun... not critical though.